ui/app/components/tool-hash.js

```diff
@@ -0,0 +1,9 @@
+import Component from '@glimmer/component';
+import { action } from '@ember/object';
+
+export default class HashTool extends Component {
```

Comment:
Might be nice to add JSDocs here and include onClear as a required attr, as well as anything else used on the template

Reply:
Great idea.